project/FrontEnd/collaborative_science_platform/lib/screens/workspace_page/mobile_workspace_page/widget/entry_card.dart

```diff
@@ -50,146 +50,164 @@ class _EntryCardState extends State<EntryCard> {
       crossAxisAlignment: CrossAxisAlignment.center,
       children: [
         Text(
-          widget.entry.isTheoremEntry ? "Theorem"
-          : widget.entry.isProofEntry ? "Proof"
-          : "",
+          widget.entry.isTheoremEntry
```

Comment:
Just a reminder, inline if elses are causing our codes to turn into spaghettis. It's OK to use this for this case, but if the logic is more complex, we may think to move this logic to inside another widget, or method.
Like

String title = theorem ?? proof

Text(title)